src/documents/Builder.md

```diff
@@ -103,6 +103,36 @@ var myBuilder = fluid.prefs.builder({
 ```
 It is not necessary to specify the primary schema; The builder will automatically find the preference specifications provided by the Framework and build a primary schema (see [Processing the Schemas](#processing-the-schemas) below for more information).
 
+#### Switching Between Native HTML and jQuery UI Widgets
```

Comment:
Yes, it was intended as a subsection. I agree it could be at the same level - figuring out where it made the most sense to put it was a bit troublesome. Thinking on it now, I think it's probably best at the same level, since it applies to any instance where an adjuster widget with textfieldSlider is used, not just the starter preferences.